working/wildcards/feature-specification.md

```diff
@@ -317,37 +317,54 @@ class C {
 
 ### Super parameters
 
-An occurrence of `super._` as a declaration of a formal parameter in a
-constructor is a compile-time error. This error also occurs in the case
-where the super parameter has an explicitly declared type and/or default
-value.
+The positional formal parameter `super._` is still allowed in generative
+constructors.
 
-*`super._` is not an error everywhere: In a method body it could be an
-invocation of an inherited getter named `_`.*
+`super._` forwards the argument's value to the super constructor invocation.
 
 ```dart
-class B {
-  final _;
-  B(this._);
+class A {
+  final int _;
```

Comment:
Doesn't have to be named `_`, most parameters won't be. The `A(...)` parameter doesn't even have to be an initializing formal.

I'd take the third example an move it up here, and only have two examples:
* One correct example which shows two `super._`s (as many correctly used features as possible)
* One with an error trying to reference `_` in the initializer list.

Maybe (incorporating all comments below):
> A positional formal parameter of the form `super._` is still allowed in non-redirecting
> generative constructors. Such a parameter forwards the argument's value to the
> super constructor invocation.
>
> ```dart
> class A {
>   final int x, y;
>   A(this.x, this.y);
> }
> class B extends A {
>   final int _;
>   B(this._, super._, super._) {
>     print(_ + x + y); // Refers to `B._`, `A.x` and `A.y`.
>   }
> }
> ```
> Similarly to `this._`, and unlike for example `super.x`,
> a `super._` does not introduce any identifier into the scope of the initializer list.
> Because of that, multiple `super._` and `this._` parameters can occur in the same
> constructor without any name collision.
> 
> ```dart
> class A {
>   final int _, v;
>   A(this._, this.v);
> }
> 
> class B extends A {
>   B(super.x, super._)
>       : assert(x > 0), // OK, `x` in scope.
>         assert(_ >= 0) // Error: no `_` in scope.
>   {
>     print(_); // OK, means `this._` and refers to `A._`.
>   }
> }
> ```

(The three examples made this a little longer than I think it needs to be, combining the examples where possible helps with that.)

Reply:
Looks great. I've taken this :)